src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs

```diff
@@ -1526,9 +1526,12 @@ private void Initialize(Uri connectionUri, WSManConnectionInfo connectionInfo)
                 // config provider.
                 SetWSManSessionOption(WSManNativeApi.WSManSessionOption.WSMAN_OPTION_USE_SSL, 1);
             }
-            if (connectionInfo.NoEncryption)
+            if (connectionInfo.NoEncryption ||
```

Comment:
After thinking about this I am uncomfortable making PSRP messages non-encrypted by default for Basic auth.  Basic auth is not recommended but I am sure many customers use it, and may rely on PSRP messages being encrypted.

Reply:
@PaulHigin: I limited the change to UNIX. PSRP/MI don't support encryption over HTTP like PSRP/WIndows/WSMan does.